examples/merge.c

```diff
@@ -220,6 +220,7 @@ static int create_merge_commit(git_repository *repo, git_index *index, merge_opt
 	check_lg2(git_repository_head(&head_ref, repo), "failed to get repo HEAD", NULL);
 	if (resolve_refish(&merge_commit, repo, opts->heads[0])) {
 		fprintf(stderr, "failed to resolve refish %s", opts->heads[0]);
+		free(*parents);
```

Comment:
You need to `free(parents)`, though, not the the first element of the `parents` array :) Note the missing `*`

Reply:
I haven't thought this low-level in awhile but I I think I get it now.

There's a reason I took "low hanging fruit" for a fix.